src/nextflow.ts

```diff
@@ -12,7 +12,7 @@ const runname: Fig.Generator = {
 
 const projectname: Fig.Generator = {
   script:
-    "find $HOME/.nextflow/assets/* -maxdepth 1 -type d | cut -d/ -f6,7 | grep / | grep -v assets",
+    "{ find * -maxdepth 0 -type f -name '*.nf' 2> /dev/null && find $HOME/.nextflow/assets/* -maxdepth 1 -type d | cut -d/ -f6,7 | grep / | grep -v assets; } 2> /dev/null",
```

Comment:
Hmm we recently changed how scripts are run, now it's run in the user's shell which means this will break for Fish users (`{ ... }` isn't valid fish syntax).

We can get around this by explicitly invoking a posix shell:
```suggestion
    `/bin/sh -c "{ find * -maxdepth 0 -type f -name '*.nf' 2> /dev/null && find $HOME/.nextflow/assets/* -maxdepth 1 -type d | cut -d/ -f6,7 | grep / | grep -v assets; } 2> /dev/null"`,
```
<br>

Or by making sure that stderr is always redirected:
```suggestion
    "find * -maxdepth 0 -type f -name '*.nf' 2>/dev/null && find $HOME/.nextflow/assets/* -maxdepth 1 -type d 2>/dev/null | cut -d/ -f6,7 | grep / | grep -v assets",
```

Reply:
also cc @fedeci, want your thoughts on this

Reply:
Windows hates /dev/null right? I'll opt for the solution that guarantees the support to more OSs.

Reply:
Honestly my experience with Windows is that it hates everything 😄 but yes I'd go for the first one myself. Looks like that's the consensus. @mribeirodantas quick fix when you're ready!

Reply:
It's done!